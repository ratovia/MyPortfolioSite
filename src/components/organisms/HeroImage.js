import React from "react"
import styled from "styled-components"
import { Photograph } from "../atoms/Photograph"

const HeroArea = styled.div`
  padding: 100px 132px;
  height: 100vh;
`
export const HeroImage = ({ imageUrl }) => {
  return (
    <HeroArea>
      <Photograph imageUrl={imageUrl} radius></Photograph>
    </HeroArea>
  )
}
