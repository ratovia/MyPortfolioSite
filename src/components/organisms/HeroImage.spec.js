import React from "react"
import renderer from "react-test-renderer"
import { HeroImage } from "./HeroImage"
import bgImage from "../../stories/assets/bg.jpg"

describe("HeroImage", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<HeroImage imageUrl={bgImage} />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
