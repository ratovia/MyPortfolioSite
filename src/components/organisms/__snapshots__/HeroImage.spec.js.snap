// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeroImage renders correctly 1`] = `
<div
  className="sc-gsDJrp jHlzdI"
>
  <img
    alt="アイキャッチ"
    className="sc-bdvvaa fWZaxb"
    height="100%"
    src="test-file-stub"
    width="100%"
  />
</div>
`;
